slice.go

```diff
@@ -7,214 +7,214 @@ import (
 	"time"
 )
 
-type SliceOfBool []bool
+type boolSlice struct {
```

Comment:
Types were renamed for better consistency with exported methods (FlagSet.BoolSlice)